pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/plan/server/ServerPlanRequestUtils.java

```diff
@@ -176,6 +176,7 @@ private static void updateQueryOptions(PinotQuery pinotQuery, OpChainExecutionCo
     Map<String, String> queryOptions = new HashMap<>(executionContext.getOpChainMetadata());
     queryOptions.put(CommonConstants.Broker.Request.QueryOptionKey.TIMEOUT_MS,
         Long.toString(executionContext.getDeadlineMs() - System.currentTimeMillis()));
+    queryOptions.put(CommonConstants.Broker.Request.QueryOptionKey.ENABLE_NULL_HANDLING, "true");
```

Comment:
this only affect V2 right?

Reply:
Yes. As we discussed, we could also decide to only do this in case the key is not already present. By doing that we would evaluate nulls by default but we could remove them in case we set `enableNullHandling` false option at query time.

Reply:
I'm going to remove this change. Lets decide whether we want to enable null handling in the leaves in a following iteration